server.go

```diff
@@ -3921,8 +3927,13 @@ func (s *server) peerConnected(conn net.Conn, connReq *connmgr.ConnReq,
 		RequestAlias:           s.aliasMgr.RequestAlias,
 		AddLocalAlias:          s.aliasMgr.AddLocalAlias,
 		DisallowRouteBlinding:  s.cfg.ProtocolOptions.NoRouteBlinding(),
-		Quit:                   s.quit,
-		AuxLeafStore:           s.implCfg.AuxLeafStore,
+		ForwardExperimentalEndorsement: func() bool {
+			return clock.NewDefaultClock().Now().Before(
```

Comment:
See: https://pkg.go.dev/time#Now "Now returns the current local time". If this go package really uses host current time of day under the hood (`man 2 gettimeofday`), you might have a discrepancy. If observation is correct, I agree it's not a big deal as it's a ~`0.73 %` deviation at worst.

Reply:
I don't think the discrepancy is important. It's not like it's a flag day consensus activation.